core/app/assets/js/ui/molecules/buttons.scss

```diff
@@ -31,12 +31,19 @@
 		color: var(--button-color);
 		padding: var(--button-padding-y) var(--button-padding-x);
 		background-color: var(--button-background-color, transparent);
+		border: $eps-border-width $eps-border-style var(--button-background-color);
```

Comment:
When the `outlined` button and the `contained` button sit side by side, the `outline` button seems bigger because the `contained` does not have a border.